group-coordinator/src/main/resources/common/message/ConsumerGroupCurrentMemberAssignmentValue.json

```diff
@@ -22,16 +22,29 @@
   "fields": [
     { "name": "MemberEpoch", "versions": "0+", "type": "int32",
       "about": "The member epoch." },
+    { "name": "PreviousMemberEpoch", "versions": "0+", "type": "int32",
+      "about": "The previous member epoch." },
```

Comment:
We need this in the case where the epoch bump does not reach the member. In this case, it will retry with the old epoch and we want to accept it.

Reply:
Sure. Updated those when relevant.